apps/angular/simple-animations/src/app/app.component.ts

```diff
@@ -17,72 +27,70 @@ import { Component } from '@angular/core';
       }
     }
   `,
-  template: `
-    <div class="mx-20 my-40 flex gap-5">
-      <section>
-        <div>
-          <h3>2008</h3>
-          <p>
-            Lorem ipsum dolor sit amet consectetur adipisicing elit. Vitae
-            mollitia sequi accusantium, distinctio similique laudantium eveniet
-            quidem sit placeat possimus tempore dolorum inventore corporis atque
-            quae ad, nobis explicabo delectus.
-          </p>
-        </div>
-
-        <div>
-          <h3>2010</h3>
-          <p>
-            Lorem ipsum dolor sit amet consectetur adipisicing elit. Vitae
-            mollitia sequi accusantium, distinctio similique laudantium eveniet
-            quidem sit placeat possimus tempore dolorum inventore corporis atque
-            quae ad, nobis explicabo delectus.
-          </p>
-        </div>
-
-        <div>
-          <h4>2012</h4>
-          <p>
-            Lorem ipsum dolor sit amet consectetur adipisicing elit. Vitae
-            mollitia sequi accusantium, distinctio similique laudantium eveniet
-            quidem sit placeat possimus tempore dolorum inventore corporis atque
-            quae ad, nobis explicabo delectus.
-          </p>
-        </div>
-      </section>
-
-      <section>
-        <div class="list-item">
-          <span>Name:</span>
-          <span>Samuel</span>
-        </div>
-
-        <div class="list-item">
-          <span>Age:</span>
-          <span>28</span>
-        </div>
-
-        <div class="list-item">
-          <span>Birthdate:</span>
-          <span>02.11.1995</span>
-        </div>
+  templateUrl: 'app.component.html',
+  animations: [
+    trigger('listAnimation', [
```

Comment:
I've used a file `app.animation.ts` for this animation in my solution, so it looks like this `animations: [LIST_ANIMATION]`. To keep the component cleaner, less lines of code.